File: 240115/특정 조건 세 정수 비교/specific-comparison-of-three-natural-numbers.py
arr = input().split()

a = int(arr[0])
b = int(arr[1])
c = int(arr[2])

array = [a,b,c]

minnum = min(array)

if minnum == a:
    print(1, end=" ")
else:
    print(0, end=" ")

if a == b == c:
    print(1)
else:
    print(0)

File: 240409/특정 조건 세 정수 비교/specific-comparison-of-three-natural-numbers.swift
import Foundation

var arr = readLine()!.split(separator: " ").map {Int($0)!}
var (a, b, c) = (arr[0], arr[1], arr[2])

if arr.min()! == a {
    print(1, terminator: " ")
} else {
    print(0, terminator: " ")
}

if a == b && a == c && b == c {
    print(1, terminator: " ")
} else {
    print(0, terminator: " ")
}